_arduino/nimbleProportionalControl.ino

```diff
@@ -0,0 +1,391 @@
+#include <NimBLEDevice.h>
+#include <Chrono.h>        // http://github.com/SofaPirate/Chrono
+#include <ESP32Encoder.h>  // https://github.com/madhephaestus/ESP32Encoder.git
+
+//  for OLED and chirps 
+#include <Arduino.h>
+#include <U8x8lib.h> // need to install u8g2 by oliver to access this library 
+#include <Wire.h>
```

Comment:
Add a README.md to this directory and add these in a list of dependencies. Then remove the comments from here.